spec/specificUtils.spec.js

```diff
@@ -155,6 +156,75 @@ describe('Specific Utils', function() {
             });
         });
 
+        describe('getCodeRef', function () {
+            it('should return promise if browser is false, promise resolve should be null', function() {
+                browser = false;
+
+                const promise = SpecificUtils.getCodeRef();
+
+                expect(promise.then).toBeDefined();
+                promise.then(function (value) {
+                    expect(value).toEqual(null);
+                });
+            });
+
+            it('should call browser.getProcessedConfig if browser is true', function() {
+                browser = jasmine.createSpyObj('browser', {
+                    'getProcessedConfig': new Promise(function() {})
+                });
+
+                SpecificUtils.getCodeRef();
+
+                expect(browser.getProcessedConfig).toHaveBeenCalled();
+            });
+
+            it('should return promise, promise resolve should be codeRef value if browser is true and browser.getProcessedConfig is successful', function() {
+                browser = jasmine.createSpyObj('browser', {
+                    'getProcessedConfig': new Promise(function(resolve) {
+                        resolve({ specs: [`C:\\Path\\test.spec.js`] });
+                    })
+                });
+                spyOn(process, 'cwd').and.returnValue('C:\\Path');
+
+                const promise = SpecificUtils.getCodeRef(0, 'testName');
+
+                expect(browser.getProcessedConfig).toHaveBeenCalled();
+                promise.then(function (codeRef) {
+                    expect(codeRef).toEqual('test.spec.js/testName');
+                });
+            });
+
+            it('should return promise, separator should be \'\\\', promise resolve should be codeRef value if browser is true and browser.getProcessedConfig is successful', function() {
```

Comment:
`separator should be \'\\\'` -> it's better to state that this function replaces separator to `/`